frontend/src/RecordingPageComp/InstrumentSelectionComp.jsx

```diff
@@ -0,0 +1,33 @@
+/* eslint-disable react/prop-types */
+
+function InstrumentSelection({ midiSounds, setSelectedInstrument }) {
+  let instrumentList = [];
+  for (
+    let i = 0;
+    i < midiSounds.current.player.loader.instrumentKeys().length;
```

Comment:
Does `midiSounds.current.player.loader.instrumentKeys()` return keys for all instruments in the API?